Userland/Libraries/LibGfx/BitmapFont.cpp

```diff
@@ -69,14 +68,13 @@ NonnullRefPtr<BitmapFont> BitmapFont::unmasked_character_set() const
     auto* new_range_mask = static_cast<u8*>(malloc(s_max_range_mask_size));
     constexpr u8 max_bits { 0b1111'1111 };
     memset(new_range_mask, max_bits, s_max_range_mask_size);
-    size_t bytes_per_glyph = sizeof(u32) * glyph_height();
-    auto* new_rows = static_cast<u8*>(kmalloc_array(s_max_glyph_count, bytes_per_glyph));
+    auto* new_rows = static_cast<u8*>(kmalloc_array(s_max_glyph_count, m_bytes_per_glyph));
     auto* new_widths = static_cast<u8*>(calloc(s_max_glyph_count, 1));
     for (size_t code_point = 0; code_point < s_max_glyph_count; ++code_point) {
         auto index = glyph_index(code_point);
         if (index.has_value()) {
             memcpy(&new_widths[code_point], &m_glyph_widths[index.value()], 1);
-            memcpy(&new_rows[code_point * glyph_height()], &m_rows[index.value() * glyph_height()], bytes_per_glyph);
+            memcpy(&new_rows[code_point * m_bytes_per_glyph], &m_rows[index.value() * m_bytes_per_glyph], m_bytes_per_glyph);
```

Comment:
Could we please split this into two, one with the fix (which I assume is here) and one with the refactoring?
this diff took far too long to understand for such a simple thing...

Reply:
I'll just push the fix to keep things dead simple. The refactor can be part of a separate PR dealing with related magic numbers.

Reply:
:shrug: sure, that's fine too